src/test/java/org/reaktivity/nukleus/http_cache/internal/streams/proxy/EdgeArchProxyIT.java

```diff
@@ -212,7 +209,7 @@ public void shouldAbortPendingOnUpdateRequestsWhenFailedPollingUpdatesAndRecover
     public void shouldCancelPushPromisesOn403() throws Exception
     {
         k3po.finish();
-        counters.assertExpectedCacheEntries(1);
+        counters.assertExpectedCacheEntries(0);
```

Comment:
Should cache proactively purge here, or just leave the cache entry intact for later eviction, to avoid unnecessary repopulation of the same cache content?